### PR TITLE
Diffusers/thuchk

### PR DESCRIPTION
**Pull request by thuc-moreh**
Modify the train python files to run with testbench repo

### CODE REVIEW

File: examples/controlnet/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-accelerate==0.19.0
+git+https://github.com/huggingface/accelerate
```
Comment (loctxmoreh) [inline examples/controlnet/requirements.txt:1]: Just curious. Why change to the git version of `accelerate`? Is there any new feature needed that are not available in 0.19.0?

File: examples/controlnet/train_controlnet_mlflow.py [hunk 2]
```diff
@@ -553,9 +553,15 @@ def parse_args(input_args=None):
...
-        default=10,
+        default=100,
         help="Log every X updates steps. Default to 10.",
     )
+    parser.add_argument(
+        "--mlflow_uri",
+        type=str,
+        default='http://127.0.0.1:5001',
+        help="URI to log mlflow.",
+    )
```
Comment (loctxmoreh) [inline examples/controlnet/train_controlnet_mlflow.py]: I think a CLI argument for mlflow tracking uri is not necessary. Just use the env var in shell script & testbench

File: examples/gpu_size_for_tasks.txt
Comment (loctxmoreh) [inline examples/gpu_size_for_tasks.txt:1]: Why is this file needed? For the bash script, i suppose?

File: examples/textual_inversion/textual_inversion_mlflow.py [hunk 3]
```diff
@@ -829,116 +829,121 @@ def main():
...
+                accelerator.log(logs, step=global_step)
+
+            if global_step >= args.max_train_steps:
+                elapsed_time = (time.time() - start_time) 
+                one_epoch_time = elapsed_time/ (args.max_train_steps / num_update_steps_per_epoch)
+                throughput = (args.max_train_steps * args.train_batch_size * args.gradient_accumulation_steps) / elapsed_time
+
+                mlflow.log_metric('avg_throughput', throughput)
+                mlflow.log_metric('one_epoch_time', one_epoch_time)
+                mlflow.log_params({'task': 'textual_inversion', 'model': args.pretrained_model_name_or_path ,'batch_size': args.train_batch_size, 'logging_interval':args.logging_steps})
```
Comment (loctxmoreh) [inline examples/textual_inversion/textual_inversion_mlflow.py:943]: I believe we only need to log this once, at the start (or end) of the run

**Review by loctxmoreh — APPROVED** [review]
Good to merge

**Review by yeomni — APPROVED** [review]
LGTM

File: examples/controlnet/requirements.txt (hunk 1 above)
Comment (thuc-moreh) [reply examples/controlnet/requirements.txt:1]: I encountered an error when using accelerate 0.19.0 with diffusers' models, even though it worked fine before. The log:
```
Traceback (most recent call last):
  File "train_controlnet_mlflow.py", line 24, in <module>
    import accelerate
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/accelerate/__init__.py", line 3, in <module>
    from .accelerator import Accelerator
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/accelerate/accelerator.py", line 33, in <module>
    from .checkpointing import load_accelerator_state, load_custom_state, save_accelerator_state, save_custom_state
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/accelerate/checkpointing.py", line 24, in <module>
    from .utils import (
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/accelerate/utils/__init__.py", line 119, in <module>
    from .megatron_lm import (
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/accelerate/utils/megatron_lm.py", line 32, in <module>
    from transformers.modeling_outputs import (
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/transformers/__init__.py", line 26, in <module>
    from . import dependency_versions_check
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/transformers/dependency_versions_check.py", line 57, in <module>
    require_version_core(deps[pkg])
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/transformers/utils/versions.py", line 117, in require_version_core
    return require_version(requirement, hint)
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/transformers/utils/versions.py", line 111, in require_version
    _compare_versions(op, got_ver, want_ver, requirement, pkg, hint)
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/transformers/utils/versions.py", line 44, in _compare_versions
    raise ImportError(
ImportError: accelerate>=0.20.3 is required for a normal functioning of this module, but found accelerate==0.19.0.
Try: pip install transformers -U or pip install -e '.[dev]' if you're working with git main
```

File: examples/controlnet/train_controlnet_mlflow.py (hunk 2 above)
Comment (thuc-moreh) [reply examples/controlnet/train_controlnet_mlflow.py]: Yeah, I will remove it

File: examples/controlnet/requirements.txt (hunk 1 above)
Comment (loctxmoreh) [reply examples/controlnet/requirements.txt:1]: ~~Looks like `accelerate>=0.20.3` is a requirements from `transformers`. Can you find out which `transformers` version demand this? A simple `pip freeze | grep transformers` in your current env would suffice.~~

Nvm, I found out. Since transformers v4.30, `accelerate==0.19.0` is no longer suffice

P/s: There is actually no issue with using the latest git version of a library. But since `transformers`, as a project, moves really fast, I believe it is beneficial for us to be aware of its requirements
